code/modules/admin/player_options.dm

```diff
@@ -193,7 +193,8 @@
 						<a href='[playeropt_link(M, "spidergib")]'>Spider</a> &bull;
 						<a href='[playeropt_link(M, "cluwnegib")]'>Cluwne</a> &bull;
 						<a href='[playeropt_link(M, "tysongib")]'>Tyson</a> &bull;
-						<a href='[playeropt_link(M, "damn")]'>(Un)Damn</a>
+						<a href='[playeropt_link(M, "damn")]'>(Un)Damn</a> &bull;
+						<a href='[playeropt_link(M, "flockgib")]'>Flock</a>
```

Comment:
```suggestion
						<a href='[playeropt_link(M, "flockgib")]'>Flock</a> &bull;
						<a href='[playeropt_link(M, "damn")]'>(Un)Damn</a>
```
having Damn at the end makes the most sense to me considering it's Not Technically a Gib

Reply:
wow that is an UGLY diff